source/preferences/troubleshoot-notifications.rst

```diff
@@ -0,0 +1,144 @@
+Troubleshoot notifications
+==========================
+
+.. include:: ../_static/badges/allplans-cloud-selfhosted.rst
+  :start-after: :nosearch:
+
+The Mattermost notifications you receive depend on your `Mattermost preferences <#check-your-mattermost-preferences>`__, the `Mattermost client <#check-your-mattermost-client-settings>`__ you're using, and the `operating system (OS) <#check-your-operating-system-settings>`__ you're running Mattermost on.
+
+If you’re having trouble receiving Mattermost notifications, see the following sections for troubleshooting steps you can follow to ensure you're receiving the notifications you want.
+
+Check your Mattermost preferences
+----------------------------------
+
+Start by ensuring that your Mattermost preferences have notifications enabled:
+
+1. Select **Settings** |gear| in the top right corner of Mattermost.
+2. Select **Desktop and mobile notifications**.
+
+If **Send notifications for** is set to **Nothing**, then Mattermost notifications are currently disabled. Select either the **All new messages** or **Mentions, direct messages, and group messages** option instead.
+
+We recommend checking your Mattermost client settings next.
+
+Check your Mattermost client settings
+-------------------------------------
+
+.. tab:: Desktop
+
+    Ensure notifications are enabled in your Mattermost server connection.
+
+    1. Select the Mattermost Server option in the top left of the desktop app, then edit the server details.
+    2. Under **Permissions**, enable **Notifications**.
+
+    [add an animated GIF here]
+
+.. tab:: Mobile
+
+    Ensure that your Mattermost mobile preferences have notifications enabled.
+
+    1. Tap on your profile picture in the bottom right corner of Mattermost.
+    2. Tap **Settings** |gear|
+    3. Tap **Notifications**.
+    4. Tap **Push Notifications**.
+    5. Ensure that **All new messages** or **Mentions, direct messages, and group messages** option is selected.
+    6. Under **Trigger push notifications when…**, select **Online, away or offline** to always receive notifications.
+
+    [animated GIF here]
+
+    Also ensure that your mobile device isn't blocking device settings. Visit the **Android** or **iOS** tab below based on your mobile device.
+
+  .. tab:: Android
+
+    Ensure that your Android device isn't blocking Mattermost notifications by granting notification permission for Mattermost in device settings.
+
+    1. Open the Android **Settings app**, and tap **Application Manager**.
+    2. Locate **Google Play Services** and enable notifications for it. 
+    3. Locate **Mattermost** and enable notifications for it. 
+
+    Also ensure that your Android device isn't set to **Do Not Disturb** mode designed to block notifications.
```

Comment:
We can potentially link Android DND docs here:
[Learn more about Do Not Disturb mode in Android.](https://support.google.com/android/answer/9069335#zippy=%2Cturn-interruptions-back-on)